frappe/utils/pdf.py

```diff
@@ -23,9 +23,9 @@
 ]
 
 
-def get_pdf(html, options=None, output: PdfWriter | None = None):
+def get_pdf(html, options=None, output: PdfWriter | None = None, letterhead: str | None = None):
 	html = scrub_urls(html)
-	html, options = prepare_options(html, options)
+	html, options = prepare_options(html, options, letterhead=letterhead)
 
 	options.update({"disable-javascript": "", "disable-local-file-access": ""})
```

Comment:
Do scripts even work because of this? 

This is disabled for security reasons https://github.com/frappe/frappe/commit/36bc812c9493f578a3a39f059252e6c967d5decc

Reply:
From the proof I added in the description, yup. Maybe the flag does not apply to headers and footers?
Because there is pre-existing js in `pdf_header_footer.html` that is used for page substitution as per https://wkhtmltopdf.org/usage/wkhtmltopdf.txt (see "Footers and Headers"). I extended the script a bit more.

Reply:
@maharshivpatel can you confirm this please.

Reply:
yup, JS works in header/footer even after disabling it.

i guess this is intentional because page substitution in header/footer uses JS.